src/impl/collection_change_builder.cpp

```diff
@@ -504,6 +526,21 @@ void CollectionChangeBuilder::move_column(size_t from, size_t to)
         std::rotate(begin(columns) + to, begin(columns) + from, begin(columns) + from + 1);
 }
 
+void CollectionChangeBuilder::set_old_partial_sync_state(partial_sync::SubscriptionState state)
```

Comment:
Why do these exist?

Reply:
I got some compiler errors when doing `old_partial_sync_state = c.old_partial_sync_state` in the `merge()` method. I was not able to resolve them any other way, but happy to change it if there is an easier way?

Reply:
What were the errors?

Reply:
Something about not being able to convert between method calls and expressions, but trying to provoke the error again, it actually compiled fine.

During development, I had issues with a faulty CCache, so perhaps these errors came from there as well. 

It appears to work now by just setting them directly, so I'll remove these methods.